src/pages/components/drawer.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Drawer
+description: >
+  A Drawer is a container for supplementary content that is anchored to the edge of a page.
+package: '@zendeskgarden/react-modals'
+components:
+  - modals/src/elements/DrawerModal/DrawerModal.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>
+        To display information or actions that are supplementary to the screen’s primary content
+      </li>
+      <li>To display a list of actions that affect the screen’s content, such as filtering data</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To block access to the primary view and force a decision from the user, use a
```

Comment:
I feel a bit unsure about this because a Drawer is actually a modal. Its component name is `DrawerModal` and it lives in the `react-modals` package. It uses the [core modal logic](https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/elements/DrawerModal/DrawerModal.tsx#L101) and is based on the [modal dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal) pattern.